write-fonts/src/tables/gvar.rs

```diff
@@ -197,19 +198,63 @@ impl GlyphVariations {
         }
     }
 
+    /// Determine if we should use 'shared point numbers'
+    ///
+    /// If multiple tuple variations for a given glyph use the same point numbers,
+    /// it is possible to store this in the glyph table, avoiding duplicating
+    /// data.
+    ///
+    /// This implementation is currently based on the one in fonttools, where it
+    /// is part of the compileTupleVariationStore method:
+    /// <https://github.com/fonttools/fonttools/blob/0a3360e52727cdefce2e9b28286b074faf99033c/Lib/fontTools/ttLib/tables/TupleVariation.py#L641>
+    ///
+    /// # Note
```

Comment:
Suggest moving the Note to an issue; buried in comments it's basically lost

Reply:
opened as #634